src/internal/packager2/layout/sbom.go

```diff
@@ -82,6 +82,18 @@ func generateSBOM(ctx context.Context, pkg v1alpha1.ZarfPackage, buildPath strin
 		}
 	}
 
+	// Stereoscope creates extra layers outside of the OCI formatted directory in the cache
```

Comment:
It seems extra layers were also getting created with Crane, but since this follows the Crane format for cached layers, we never noticed. https://github.com/anchore/stereoscope/issues/387